lib/Dialects/LinalgExt/Transforms/TilingExternalModels.cpp

```diff
@@ -103,16 +103,40 @@ struct LinalgOpTilingInterface
                          ArrayRef<OpFoldResult> sizes,
                          bool tileDestOperands) const {
     LinalgOp linalgOp = cast<LinalgOp>(op);
+    if (op->getNumResults() != 1) {
+      // TODO: Need a failure message here, but `notifyMatchFailure` is only a
+      // method on `PatternRewriter`.
+      return {};
```

Comment:
Looks like this method should return `FailureOr<SmallVector<Operation*>>`

Reply:
Agreed. I think the interface needs to be updated. Ill do that in a follow up.